tutorials/understanding-gov/index.md

```diff
@@ -0,0 +1,345 @@
+---
+parent:
+title: Understand the Gov Module
+order: 0
+description: Use the Cosmos SDK gov module for participation in Cosmos SDK governance. Create proposals of any message type thanks to the new gov module.
+tags:
+  - tutorial
+  - cosmos-sdk
+  - dev-ops
+---
+
+# Understand the Gov Module
+
+The [`gov`](https://docs.cosmos.network/v0.46/modules/gov) module enables governance on Cosmos SDK. It allows you to create proposals of any message type, and vote on them.
+
+## Usage of the gov module
+
+When the gov module is enabled on a chain (for example the Cosmos Hub), the users can submit a proposal to be voted on by the community.
+
+A proposal can be an upgrade of the chain, a change of the parameters of the chain, a simple text proposal, or any other message type. This tutorial will focus on how you can participate in governance, by creating and voting on proposals.
+
+Before starting, review some terminology:
+
+* **Proposal:** A proposal is a suggestion that is submitted to the network for voting. Once a proposal is submitted, it is identified by a unique proposal ID.
+
+* **Message:** A proposal includes an array of `sdk.Msgs` which are executed automatically if the proposal passes. This means you can submit a proposal about any action on which the governance module has `authority`.
```

Comment:
Because authority is actually a field, you can fill yourself in a message: e.g. https://github.com/cosmos/cosmos-sdk/blob/5deb137f7a83506381472928d898cd823735aade/proto/cosmos/upgrade/v1beta1/tx.proto#L34.
Many modules set governance has the authority.